pkgs/by-name/qb/qbittorrent/package.nix

```diff
@@ -0,0 +1,93 @@
+{
+  apple-sdk_15,
+  boost,
+  cmake,
+  dbus,
+  fetchFromGitHub,
+  guiSupport ? true,
+  hicolor-icon-theme,
+  lib,
+  libtorrent-rasterbar,
+  openssl,
+  pkg-config,
+  python3,
+  qt6,
+  stdenv,
+  trackerSearch ? true,
+  webuiSupport ? true,
+  zlib,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "qbittorrent" + lib.optionalString (!guiSupport) "-nox";
+  version = "5.0.2";
+
+  src = fetchFromGitHub {
+    owner = "qbittorrent";
+    repo = "qBittorrent";
+    rev = "release-${finalAttrs.version}";
+    hash = "sha256-JIURzAkVNYjHAx8yY0WaKNK4N/z9ndqjk0EXkJH9AzY=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+    qt6.wrapQtAppsHook
+  ];
+
+  buildInputs =
+    [
+      boost
+      hicolor-icon-theme
+      libtorrent-rasterbar
+      openssl
+      qt6.qtbase
+      qt6.qtsvg
+      qt6.qttools
+      zlib
+    ]
+    ++ lib.optionals guiSupport [ dbus ]
+    ++ lib.optionals (guiSupport && stdenv.hostPlatform.isLinux) [ qt6.qtwayland ]
+    ++ lib.optionals trackerSearch [ python3 ]
+    ++ lib.optionals stdenv.hostPlatform.isDarwin [ apple-sdk_15 ];
```

Comment:
My assumption was something from the sdk was still needed because of:
https://github.com/NixOS/nixpkgs/blob/ad62813bfe59e0cf31b58df0a82201481ee371e2/pkgs/applications/networking/p2p/qbittorrent/default.nix#L54-L55
Am I wrong? If the default from stdenv is enough, why was Cocoa added?

Reply:
So shouldn't the newer apple-sdk_15 be fine as well? or is it too new for x86_64-darwin?
Let's see if ofborg builds and if not, I'll downgrade to apple-sdk_11 since on aarch64-darwin it built and ran just fine with the newer sdk even if not on macos 15.